nomad/structs/structs.go

```diff
@@ -2959,10 +2977,10 @@ func (tg *TaskGroup) Validate(j *Job) error {
 	// Validate the migration strategy
 	switch j.Type {
 	case JobTypeService:
-		if tg.Count == 1 && tg.Migrate != nil {
-			mErr.Errors = append(mErr.Errors, fmt.Errorf("Task Group %v should not have a migration strategy with a count = 1", tg.Name))
-		} else if err := tg.Migrate.Validate(); err != nil {
-			mErr.Errors = append(mErr.Errors, err)
+		if tg.Migrate != nil {
```

Comment:
Why did you remove this?

Reply:
I don't like the UX of it as user scales up and down a count they will get hit by this.

Reply:
I guess it doesn't matter since MaxParallel has to be >0, so Count==1 should Just Work whether or not we special case it.